README.md

```diff
@@ -34,19 +45,29 @@ $$
     raleos = [6, 11, 2] # thinning policies in the years 6, 8, 10.
     cosechas = [18, 29, 3] # harvesting policies in the years 18, 21, 24, ... (every 3 years)
     # all feasible histories combining thinning and harvesting policies will be generated
+    min_ral = 6 #minimun age where you can thinn a tree
```

Comment:
![image](https://github.com/user-attachments/assets/8a292e93-b786-40b0-b9c1-bd9be522e3f2)
# lower bound to tree thinning actions